management-registry/src/test/java/org/terracotta/management/registry/ManagementRegistryTest.java

```diff
@@ -67,4 +76,11 @@ public ContextContainer getContextContainer() {
     }
   }
 
+  public static abstract class CapabilityContextMixin {
```

Comment:
nice, that's very nice of you to provide an example in the test.
I'll remember that there is a simple example provided in a test